### PR TITLE
Multiranges docs update.

### PR DESCRIPTION
**Pull request by vpetrovykh**
This PR needs to be updated with the correct REPL snippets once https://github.com/edgedb/edgedb-cli/issues/1149 is done.

Other than that it's ready so please review.

### CODE REVIEW

File: docs/changelog/4_x.rst [hunk 1]
```diff
@@ -205,13 +205,22 @@ fields:
...
 -----------
 
-We've made it easier to work with ranges by adding ``multirange`` as one of
-the datatypes. Multiranges consist of one or more ranges and allow expressing
-intervals that are not contiguous. Multiranges are automatically normalized to
-contain non-overlapping ranges that are ordered according to their boundaries.
-All the usual range functions like ``overlaps`` or ``contains`` work with any
-combination of ranges and multiranges providing more flexibility in
-expressions.
+We've made it easier to work with ranges by adding :ref:`multirange
```
Comment (msullivan) [inline docs/changelog/4_x.rst:208]: Suggestion: by adding a multirange datatype

File: docs/changelog/4_x.rst [hunk 2]
```diff
@@ -205,13 +205,22 @@ fields:
...
+<ref_std_range>` as one of the datatypes. Multiranges consist of one or more
+ranges and allow expressing intervals that are not contiguous. Multiranges are
+automatically normalized to contain non-overlapping ranges that are ordered
+according to their boundaries. All the usual range operators and functions
+like ``overlaps`` or ``contains`` work with any combination of ranges and
+multiranges providing more flexibility in expressions.
```
Comment (msullivan) [inline docs/changelog/4_x.rst:214]: multiranges, providing

File: docs/stdlib/range.rst [hunk 3]
```diff
@@ -105,61 +105,61 @@ cast into a :eql:type:`json` structure:
...
 
 .. code-block:: edgeql-repl
 
-    db> select <range<int64>>to_json('{
-    ...   "lower": 1,
-    ...   "inc_lower": true,
-    ...   "upper": 10,
-    ...   "inc_upper": false
-    ... }');
+    edgedb> select <range<int64>>to_json('{
```
Comment (Dhghomon) [inline docs/stdlib/range.rst:116]: Style guide note: db> and ... are used for edgeql-repl blocks (keeps output as short as possible which is nice for mobile)

**Comment by vpetrovykh** [comment]
The CLI snippets have been updated with the latest version and no longer rely on JSON representation.

**Review by aljazerzen — APPROVED** [review]

**Pull request merged**